utils/env_preferences/src/apple.rs

```diff
@@ -59,6 +59,16 @@ fn get_string(ptr: CFStringRef) -> Result<String, RetrievalError> {
     }
 }
 
+/// Retrieves system locales for Apple operating systems, in the order preferred by the
+/// user, it consumes [`CFLocaleCopyPreferredLanguages`](https://developer.apple.com/documentation/corefoundation/1542887-cflocalecopypreferredlanguages)
+/// to copy the languages prefered by the user.
+///
+/// ### Example
+///
+/// ```rust
+/// use env_preferences::get_locales;
+/// let locales: Vec<String> = get_locales().unwrap();
```

Comment:
this is not a useful example, it doesn't show anything that isn't obvious from the function signature

Reply:
An idea could be to format digits on the basis of locale (default settings)?

Reply:
just remove it for now